mkdocs/themes/mkdocs/css/base.css

```
@@ -42,32 +44,41 @@ body > .container {
     display: inline-block;
     padding: 4px;
     line-height: 1.428571429;
-    background-color: #fff;
-    border: 1px solid #ddd;
+    /* csslint ignore:start */
+    background-color: var(--bs-secondary-bg-subtle);
+    border: 1px solid var(--bs-secondary-border-subtle);
+    /* csslint ignore:end */
     border-radius: 4px;
     margin: 20px auto 30px auto;
 }
 
 h1 {
-    color: #444;
+    /* csslint ignore:start */
+    color: var(--bs-secondary-text-emphasis);
+    /* csslint ignore:end */
     font-weight: 400;
     font-size: 42px;
 }
 
 h2, h3, h4, h5, h6 {
-    color: #444;
+    /* csslint ignore:start */
+    color: var(--bs-secondary-text-emphasis);
+    /* csslint ignore:end */
     font-weight: 300;
 }
 
 hr {
     border-top: 1px solid #aaa;
+    opacity: 1;
 }
 
 pre, .rst-content tt {
     max-width: 100%;
-    background: #fff;
-    border: solid 1px #e1e4e5;
-    color: #333;
+    /* csslint ignore:start */
+    background: var(--bs-secondary-bg-subtle);
+    border: solid 1px var(--bs-secondary-border-subtle);
+    color: var(--bs-secondary-text-emphasis);
```

The colors have super little contrast - just 3 shades of grey. This also deviates from the prior theme.

And it deviates very far from highlightjs which has high-contrast colors - in places where it does not kick in, colors will randomly be grey.

I have devised that these colors match the prior colors very well, as well as being close to highlightjs:

```suggestion
    background-color: var(--bs-body-bg);
    border: solid 1px var(--bs-border-color);
    color: var(--bs-body-color);
```


And it seems that you've noticed this effect yourself, because you replaced `nohighlight` with `text` to avoid this

---

> And it seems that you've noticed this effect yourself, because you replaced `nohighlight` with `text` to avoid this

I did, but I would have actually preferred the colors I chose over highlight.js colors. In fact, I actually considered setting `!important` on those styles so they override highlight.js. But then I realized a user may prefer a different highlight.js theme which those colors would not work with.

---

Applied this suggestion in https://github.com/mkdocs/mkdocs/commit/351b87939ddc61bc767e2b869e2a2673fe9af5ae